lms/static/images/icon-cert-white.svg

```diff
@@ -0,0 +1,10 @@
+<svg id="Layer_2" data-name="Layer 2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12.85 9.22">
+    <defs>
+        <style>.cls-1{fill:#fff;}</style>
+    </defs>
+    <title>cert-icon</title>
```

Comment:
@AlasdairSwan is this element necessary to begin with? I was thinking about removing it.